tcms/testcases/templates/testcases/search.html

```diff
@@ -113,6 +113,11 @@
                 <input id="id_tag" type="text" class="form-control" value="{{ form.tag__name__in.value|default:'' }}">
                 <p class="help-block">{% trans "Separate multiple values with comma (,)" %}</p>
             </div>
+
+            <label class="col-md-1 col-lg-1" for="id_text">{% trans "Text" %}</label>
+            <div class="col-md-3 col-lg-3">
+                <input id="id_text" type="text" class="form-control" value="">
```

Comment:
Either use `placeholder` attribute if you want to give a hint to the user, or remove empty value attribute.

Reply:
I'll remove it to avoid problems with the translation. There is no phrase like "Enter the text" in the translation files, and "Text" is the title of the field.